examples/tutorial_4_streamproto2.md

```diff
@@ -0,0 +1,106 @@
+In the [previous tutorial](tutorial_3_streamproto.md), we learned how to create a custom streaming protocol.
+
+This tutorial is very similar to the [previous tutorial](tutorial_3_streamproto.md) in that a stream of data will be consumed, however in this tutorial we will learn how to use alternative libp2p streaming semantics.
+
+# Custom protocol in libp2p
+Let's create a `part4.nim`, and import our dependencies:
+```nim
+import bearssl
+import chronos
+import stew/byteutils
+
+import libp2p
+```
+This is similar to the third tutorial. Next, we'll declare our custom protocol.
+```nim
+const TestStreamCodec = "/test/stream-proto/1.0.0"
```

Comment:
```suggestion
const TestStreamCodec = "/test/stream-proto/2.0.0"
```
Just to differentiate the two protocols :)

Reply:
Good idea, done.